app/components/stripe-cc-form/component.js

```diff
@@ -0,0 +1,60 @@
+import Ember from 'ember';
+import { createStripeToken } from 'diesel/utils/stripe';
+
+export default Ember.Component.extend({
+  store: Ember.inject.service(),
+  error: null,
+
+  name: null,
+  number: null,
+  cvc: null,
+  expMonth: null,
+  expYear: null,
+  zip: null,
+
+  isSaving: false,
+
+  organization: null,
+  plan: Ember.computed.reads('organization.plan'),
+
+  actions: {
+    saveCC() {
+      let {
+        name,
+        number,
+        cvc,
+        zip,
+        plan,
+        organization,
+        expMonth: exp_month,
+        expYear: exp_year
+      } = this.getProperties(
+        ['name', 'number', 'cvc',
+         'expMonth', 'expYear', 'zip',
+         'organization', 'plan']);
+
+      let stripeOptions = { name, number, cvc, exp_month, exp_year, zip };
+
+      this.set('isSaving', true);
+      this.set('error', null);
+      let success = false;
+      createStripeToken(stripeOptions).then(({id: stripeToken}) => {
```

Comment:
destructuring arguments — stripe responds with a `{id: 'some-token'}` object, this assigns the id property to a local variable `stripeToken` that can be used in the function.